scipy/optimize/_constraints.py

```diff
@@ -249,6 +250,57 @@ def __init__(self, constraint, x0, sparse_jacobian=None,
         self.bounds = (lb, ub)
         self.keep_feasible = keep_feasible
 
+    def is_feasible(self, x):
+        """
+        The truth of whether a given vector of independent variables
+        satisfies the constraint.
+
+        Parameters
+        ----------
+        x : array-like
+            Vector of independent variables.
+
+        Returns
+        -------
+        feasible : bool
+            Whether the vector of independent variables satisfies the 'm'
```

Comment:
Mentioning 'm' here looks a bit odd. Maybe just "... satisfies the constraint." or "... all the constraints."